### ISSUE TITLE
Redis connection

### ISSUE THREAD

**Issue by jaeming** [raulanatol/camaleon-media_aws_s3#2]
Is there any documentation anywhere on this plugin? I added it and it seems it requires redis which I've not used before. I installed redis gem and redis-server on my dev machine and now I can browse the s3 bucket from the file upload. However I'm not able to upload images, nor use any of the existing images in the bucket.  I'd love to know what config or redis integration made this work for you. Thanks!

UPDATE: it does actually upload images. I went to the actual bucket to confirm. But the file browser does not refresh or show what was uploaded. Also, regardless, I still can not select and image for use, even if I can see it. When I doubleclick it, the rails console says: `ActionController::RoutingError (No route matches [GET] "/admin/uploads/benji/uploads/image/file/10/53ffe629a584c663122b7f6e_blur.jpg"):`

So it appears it doesn't know the image should be coming from s3 and thinks it's from locally...I'm still investigating why.

**Comment by raulanatol** [comment]
Hi @jaeming!

Sorry for the little documentation. 
I'm working to upgrade.

The cache configuration is in the /app/apps/plugins/media_aws_s3/models/cache_connector.rb file.
You can specify your preferred cache system. This plugin are using https://rubygems.org/gems/cache gem to wrap caches.

For example if you can use Rails.cache:

```
class Plugins::MediaAwsS3::Models::CacheConnector < ElFinderS3::CacheConnector

 def initialize
 super(Rails.cache)
 end

end
```

If you can show more information: https://github.com/raulanatol/el_finder_s3

About the _no route matches routes_ error please verify if the CDN Base path URL is similar like: 

```
https://YOUR_BUCKET_NAME.s3.amazonaws.com
```

Regards.

**Comment by jaeming** [comment]
Thank you, I didn't know I could swap out for the Rails cache there. I did that I don't need redis now. I also don't have the routing error anymore but still somehow I'm unable to use the images I can bowse on s3. I see the file names and no thumbnail. I click get info and see the file size but when I click on the image itself, the import button never appears. Also, after uploading, I don't see what I uploaded.  I don't think the problem is related to your gem and probably my inexperience comes into play here so I'll go ahead and close the issue and keep playing around with it.  Thanks so much for your help.
